test/e2e/e2e_test.go

```diff
@@ -0,0 +1,70 @@
+package e2e_test
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/kubev2v/migration-planner/api/v1alpha1"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("e2e", func() {
+
+	var (
+		svc      PlannerService
+		agent    PlannerAgent
+		sourceId string
+		agentIP  string
+		err      error
+		systemIP = os.Getenv("PLANNER_IP")
+	)
+
+	BeforeEach(func() {
+		svc, err = NewPlannerService(defaultConfigPath)
+		Expect(err).To(BeNil())
+		agent, err = NewPlannerAgent(defaultConfigPath, vmName)
+		Expect(err).To(BeNil())
+		sourceId, err = svc.Create("testsource")
+		Expect(err).To(BeNil())
+		Expect(sourceId).ToNot(BeNil())
+		err = agent.Run(sourceId)
```

Comment:
Greate work!
I see that agent.Run creates a  VM but I can't find where do we remove it?

Reply:
In AfterEach method we call 'agent.Remove()' that will remove the VM